src/index.js

```diff
@@ -13,6 +13,10 @@ import Armory from './armory/Armory.js';
 import TrainingArena from './trainingarena/TrainingArena.js';
 import Battleground from './battleground/Battleground.js';
 import ConfirmEmail from './login/ConfirmEmail.js';
+import Cookies from 'universal-cookie';
+
+const cookie = new Cookies();
+const loggedIn: boolean = (cookie.get('token')) ? true : false;
```

Comment:
You also need to make sure the token is valid. If you log on with a second computer the token won't be valid, but it will exist.

Reply:
You and baylor have to implement some sort of verifyToken api call to do this